_posts/2024-09-12-opensearch-project-roadmap-2024-2025.md

```diff
@@ -0,0 +1,187 @@
+
+### Roadmap Theme 9: Releases and project health
+
+With contributions ranging from code enhancements to feature requests across all roadmap themes, the OpenSearch Project community is working together on maintaining the stability of the codebase while ensuring CI/CD pipelines remain green across all active branches. This provides a reliable foundation for both new and existing contributors, reduces bugs, and safeguards feature integrity. The key repository health metrics are publicly available on the [Ops Dashboard](https://metrics.opensearch.org/_dashboards/app/dashboards#/view/f1ad21c0-e323-11ee-9a74-07cd3b4ff414?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-4y,to:now))&_a=(description:'OpenSearch%20Ops%20Metrics',filters:!(),fullScreenMode:!f,options:(hidePanelTitles:!f,useMargins:!t),query:(language:kuery,query:''),timeRestore:!t,title:'OpenSearch%20Ops%20Metrics',viewMode:view)).
+
+The OpenSearch [release process](https://github.com/opensearch-project/.github/blob/main/RELEASING.md) is fully automated, including a one-click release system for some of the products like OpenSearch Benchmarks. Each product adheres to [semantic versioning (semver)](https://opensearch.org/blog/what-is-semver/), ensuring that breaking changes only occur in major versions. Releases follow a structured [schedule](https://opensearch.org/releases.html#release-schedule), starting with a code freeze and release candidate generation, and are driven by automated workflows that eliminate the need for manual sign-offs. We’re also building a [Central Release Dashboard](https://opensearch.org/release-dashboard) to streamline and provide visibility into the release pipeline from start to finish. 
+
+
+## Get involved
+
+We recognize that community engagement is crucial to the success of all the futuristic innovations that are mentioned in this post. We invite the open-source community to review our roadmap, provide feedback, and contribute to the OpenSearch Project. Your insights and contributions will be invaluable in helping us achieve these goals and continue improving OpenSearch. 
+
+You can [propose new ideas and features](https://github.com/opensearch-project/.github/blob/main/FEATURES.md) at any time using a GitHub issue and following our [feature request template](https://github.com/opensearch-project/.github/blob/main/.github/ISSUE_TEMPLATE/FEATURE_REQUEST_TEMPLATE.md). Once proposed, they can be included in the [public roadmap](https://github.com/orgs/opensearch-project/projects/206/views/11) by adding corresponding labels (such as Meta, RFC, and Roadmap), which are automatically populated for all the repositories and are categorized by themes for clarity. If you have any questions or suggestions on improving our processes, please feel free to reach out or contribute directly through [GitHub](https://github.com/opensearch-project). 
+
```

Comment:
Above: "Once proposed, the issue can be included"? Or "Once proposed, the idea or feature can be included"? (What does "They" refer to?)